tests/db-client/DBClient.ts

```diff
@@ -0,0 +1,21 @@
+import { connect, Client } from 'ts-postgres';
+import 'dotenv/config';
+
+export class DBClient {
+    protected client: Client;
+
+    constructor(client: Client) {
+        this.client = client;
+    }
+
+    public static async createAndConnectClient() {
```

Comment:
For me it seems like we'd better have static getter for client. So getter will resolve this connection code. Something like this: 
```
class Client {
private static client: dbClient | undefined;
private static async getClient(): dbClient {
    	if (!this.client)
		this.client = await connect({ ...props });
	return this.client;
}
static query(query: string): Promise<shotatam> {
	const client = await this.getClient();
	return await client.query(query);
} 

```